### PR TITLE
feat product listing page

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the product listing page
Note added some basic utils in ecom but will move them to CL in future.

https://user-images.githubusercontent.com/75560307/154323215-c26badc1-9792-47d7-95d4-15b3d935fdc2.mp4

### CODE REVIEW

File: CSS/product-list-page.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    justify-content: space-between;
+    padding: 0 2rem;
+}
+.filter-main-section{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    padding: 2rem;
+}
+@media only screen and (max-width: 800px) {
```
Comment (sakleprajwal) [inline CSS/product-list-page.css:32]: The ideal max-width for a mobile screen is 576px as per my knowledge. You can do your own research.

**Review by sakleprajwal — APPROVED** [review]
Suggest a minor change. Everything else looks good. Please go ahead.

File: pages/product-listing-page.html [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+                        </div>    
+                    </div>
+                    <div class="footer-card">
+                            <a href="#" target="_blank"><div class="icons">Order soon</div></a>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="header maple-flex">
+                        <div class="header-image-text-div">
+                                <img class="header-image" src="../Assets/coffee-pro.jpg" alt="" srcset="">
```
Comment (Sameony) [inline pages/product-listing-page.html]: Dont leave alt and srcset empty

**Review by Sameony — COMMENTED** [review]
do fill up the alt and srcset values before merging

**Review by tanishq20 — APPROVED** [review]
Code looks good and clean.

**Review by coldpigli — APPROVED** [review]
Everything looks good. Just one suggestion. Try to use more semantic tags such as <section> article etc instead of using <div> everywhere.

File: CSS/product-list-page.css (hunk 1 above)
Comment (krishanu2911) [reply CSS/product-list-page.css:32]: Thank you for suggesting prajwal but i was following that mine app is breaking that this point so using 800px as vikram also said to customize for break points ACC. to your own app bzc every app behave differently so I better to see that where exactly your app is breaking

File: pages/product-listing-page.html (hunk 2 above)
Comment (krishanu2911) [reply pages/product-listing-page.html]: Done

**Pull request merged**